packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -31,3 +31,51 @@ export const userLogPayloadGuard = z.object({
 });
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
+
+export const signingKeyPayload = z.object({
+  keyId: z.string(),
+});
+
+export const signingKeyManagementPayloadGuard = z.object({
+  signing_algorithm: z.enum(['RS256']),
+  signing_keys: z.object({
+    previously_used_keys: signingKeyPayload.array(),
+    currently_used_key: signingKeyPayload,
+    next_key_in_queue: signingKeyPayload,
+  }),
+});
```

Comment:
looks cool, but do we have specific API design for this data? cc @simeng-li @wangsijie

Reply:
I've got a design draft for this, I can organize the draft for discussion/review. Or we can leave an issue to do this in the later version as this is not urgent for v1.

Reply:
cool, you can drive the discussion if needed. we can remove this part for now.

Reply:
Issue added for later discussion.
(Ref. LOG-1277)